### PR TITLE
Added github actions to check Terraform code

### PR DESCRIPTION
**Pull request by Kirill-Kulikov**
Added github actions to implement automatic verification of Terraform code, at the stage of checking PR and committing to all branches, performs steps:  
- `terraform-init` - Init and validate Terraform code (terraform init, terraform validate)
- `terraform-format` - Check Terraform code format (terraform fmt)
- `terraform-tflint` - Find possible problems with Terraform code linter (tflint)

Additionally formatted pritunl module `aws-ec2-pritunl/efs.tf` according to terraform format code to pass the `terraform-format` stage.

### CODE REVIEW

File: .github/workflows/terraform-ci.yml [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+on: [push, pull_request]
+
+defaults:
+  run:
+    shell: sh
+
+jobs:
+  # Terraform init and validate configuration
+  terraform-init:
```
Comment (p1gmale0n) [inline .github/workflows/terraform-ci.yml:11]: the main purpose of this stage - terraform validate
it seems to me that it should be renamed

File: .github/workflows/terraform-ci.yml [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+    name: 'Terraform-init'
+    runs-on: ubuntu-latest
+    environment: dev
```
Comment (p1gmale0n) [inline .github/workflows/terraform-ci.yml:14]: i don't really understand what the environment is used for here

File: .github/workflows/terraform-ci.yml [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+    container:
+      image: hashicorp/terraform:0.14.6
```
Comment (p1gmale0n) [inline .github/workflows/terraform-ci.yml:16]: maybe it makes sense to always use the latest version of terraforms for checks?

**Review by p1gmale0n — CHANGES_REQUESTED** [review]

File: .github/workflows/terraform-ci.yml (hunk 1 above)
Comment (Kirill-Kulikov) [reply .github/workflows/terraform-ci.yml]: renamed
https://github.com/maddevsio/aws-eks-base/pull/29/commits/2fe5c46e524ce9fc6478306146fcc1d7b2c98a26

File: .github/workflows/terraform-ci.yml (hunk 2 above)
Comment (Kirill-Kulikov) [reply .github/workflows/terraform-ci.yml]: Deleted
https://github.com/maddevsio/aws-eks-base/pull/29/commits/2fe5c46e524ce9fc6478306146fcc1d7b2c98a26

File: .github/workflows/terraform-ci.yml (hunk 3 above)
Comment (Kirill-Kulikov) [reply .github/workflows/terraform-ci.yml:15]: Please describe distinct Issue for it

Comment (Kirill-Kulikov) [reply .github/workflows/terraform-ci.yml:15]: Created draft issue, please check it
https://github.com/maddevsio/aws-eks-base/issues/35

**Review by p1gmale0n — APPROVED** [review]

**Review by halfb00t — APPROVED** [review]

**Pull request merged**